charts/datadog/values.yaml

```diff
@@ -283,6 +283,10 @@ datadog:
     ## ref: TODO - add doc link
     enabled: false
 
+  networkConfig:
```

Comment:
Have you considered to add `systemProbe.enableNetwork` (`true` by default for backward compatibility) instead of created a new section?

Reply:
There is a network_config section in the agent config, where new config will eventually be added. So I figured adding a new "namespace" was better for future compatibility.